src/Common/ErrorCodes.cpp

```diff
@@ -561,6 +561,7 @@
     M(591, SQLITE_ENGINE_ERROR) \
     M(592, DATA_ENCRYPTION_ERROR) \
     M(593, ZERO_COPY_REPLICATION_ERROR) \
+    M(594, PARTITION_BY_NOT_SUPPORTED) \
```

Comment:
```suggestion
    M(594, NOT_IMPLEMENTED) \
```

Reply:
And I also cannot commit this via web interface. You can apply three suggestion or cherry pick https://github.com/ClickHouse/ClickHouse/commit/9c1cb601e957f96e5bbceb811bb0070060a63d5a

Reply:
Pushed existing `NOT_IMPLEMENTED`.